com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "com.rustdesk.RustDesk",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "rustdesk",
+    "icon": "share/icons/hicolor/scalable/apps/rustdesk.svg",
+    "modules": [
+      "shared-modules/libappindicator/libappindicator-gtk3-12.10.json",
+      "xdotool.json",
+      {
+        "name": "pam",    
+        "buildsystem": "simple",
+        "build-commands": [
+            "./configure --disable-selinux --prefix=/app && make -j4 install"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz",
+            "sha256": "eff47a4ecd833fbf18de9686632a70ee8d0794b79aecb217ebd0ce11db4cd0db"
+          }
+        ]
+      },
+      {
+        "name": "libarchive",
+        "buildsystem": "autotools",
+        "config-opts": ["--disable-static"],
+        "cleanup": ["/include", "/lib/*.la", "/lib/pkgconfig", "/share"],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz",
+                "sha256": "9015d109ec00bb9ae1a384b172bf2fc1dff41e2c66e5a9eeddf933af9db37f5a"
+            }
+        ]
+      },
+      {
+        "name": "rustdesk",
+        "buildsystem": "simple",
+        "build-commands": [
+          "/app/bin/bsdtar -zxvf rustdesk*.deb",
+          "rm -rf /app/bin",
+          "tar -xvf ./data.tar.xz",
+          "cp -r usr/*  /app/",
+          "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk",
+          "mv /app/share/applications/rustdesk.desktop /app/share/applications/com.rustdesk.RustDesk.desktop",
+          "mv /app/share/applications/rustdesk-link.desktop /app/share/applications/com.rustdesk.RustDesk-link.desktop",
+          "sed -i '/^Icon=/ c\\Icon=com.rustdesk.RustDesk' /app/share/applications/*.desktop",
+          "install -Dm644 usr/share/icons/hicolor/256x256/apps/rustdesk.png /app/share/icons/hicolor/256x256/apps/com.rustdesk.RustDesk.png"
+        ],
+        "cleanup": ["/include", "/lib/pkgconfig", "/share/gtk-doc", "/share/man"],
+        "sources": [
+          {
+            "type": "file",
+            "only-arches": ["aarch64"],
+            "url": "https://github.com/rustdesk/rustdesk/releases/download/1.2.4/rustdesk-1.2.4-aarch64.deb",
```

Comment:
why not building from source?

Reply:
why wasting electric power on the earth if there is a deb while it is building from source.

Reply:
because it might not have been build against the right dependencies

because it's the prefered method.

You argument is irrelevant.

Reply:
Also it make respecting AGPL-3.0 harder since flathub end up not having the sources.

Reply:
I do not agree with you.

Reply:
Are there any technical issues with building from source? Your debs are produced against the ABI/API of the system that runs on your CI. The ABI/API used by the runtimes here are completely different

Reply:
No ABI/API problem, we have flatpak release here, many download every day. https://github.com/rustdesk/rustdesk/releases/tag/nightly

Reply:
Since this seems to use flutter and building flutter from source in Flathub is not possible I'm marking it as solved.

Reply:
Thanks